autopep8 related changes

quaquel opened the pull request:
I ran autopep8 to clean up the code. I might do some work on the plane to make all code fit into 79 characters.

jdherman commented:
Haha, you don't waste any time!! Thanks @quaquel 

A couple of questions/comments:
- Since I haven't set up unit tests yet, could you confirm that the example still runs? (I assume that it will, it looks like none of the changes are major).
- Let's .gitignore and remove the dotfiles generated by your IDE. 
- It looks like the majority of lines changed are only indentation. Is it really required by PEP8 to have 4 spaces of indentation instead of 2? I really prefer 2. This would help with the 80-character width, too.

That's all, then we can merge. Thanks again!

quaquel commented:
I have updated gitignore and removed the eclipse files. I made some more layout changes related to pep8. The example still runs fine. 

pep8 specifies 4 spaces: https://www.python.org/dev/peps/pep-0008/#indentation. Line length should be 79 for code and 72 for docstrings. This I stick to less closely.

jdherman commented:
Ok let's do it. Thanks!
